pvr.hts/resources/settings.xml

```diff
@@ -8,7 +8,7 @@
     <setting id="user" type="text" label="30004" default="" />
     <setting id="pass" type="text" label="30005" option="hidden" enable="!eq(-1,)" default="" />
     <setting id="connect_timeout" type="slider" option="int" range="1,60" label="30006" default="10" />
-    <setting id="response_timeout" type="slider" option="int" range="1,60" label="30007" default="5" />
```

Comment:
This seems to have been accidentally removed

Reply:
oops, thx